VAMobile/src/screens/HealthScreen/HealthScreen.tsx

```diff
@@ -43,10 +44,26 @@ export const HealthScreen: FC<HealthScreenProps> = ({ navigation }) => {
   const { prescriptionsNeedLoad } = useSelector<RootState, PrescriptionState>((s) => s.prescriptions)
   const { data: userAuthorizedServices } = useAuthorizedServices()
 
-  const onAppointments = navigateTo('Appointments')
-  const onSecureMessaging = navigateTo('SecureMessaging')
-  const onVaVaccines = navigateTo('VaccineList')
-  const pharmacyNavHandler = navigateTo('PrescriptionHistory')
+  const onAppointments = () => {
+    if (waygateNativeAlert('WG_Appointments')) {
+      navigateTo('Appointments')
```

Comment:
```suggestion
      navigateTo('Appointments')()
```

Reply:
I had these changes in my local branch, just didn't push them up. Should be updated now ✅